packages/react-components/components/SelectionOption.tsx

```diff
@@ -33,7 +32,7 @@ export default function SelectionOption({
           <View style={styles.iconContainer}>
             {/* TODO: make proper checkbox */}
             {!isSelected && <View style={styles.circle} />}
-            {isSelected && <CheckCircle />}
+            {isSelected && <RadioButton />}
```

Comment:
It looks like you forgot to import `RadioButton` into this file

Reply:
thanks!